src/urllib3/poolmanager.py

```diff
@@ -419,6 +419,16 @@ def urlopen(  # type: ignore[override]
         """
         u = parse_url(url)
 
+        if u.scheme is None:
+            warnings.warn(
+                "URLs without a scheme (ie 'https://') are deprecated and begin raising an error "
```

Comment:
nit: `...are deprecated and will raise an error in a future version of urllib3`

Reply:
Done ✔️